src/preact.js.flow

```diff
@@ -0,0 +1,11 @@
+/* @flow */
+
+import * as React from 'react';
+
+export * from 'react';
+
+const Preact = Object.assign({}, React, {
```

Comment:
Usually we lowercase, but I don't imagine it actually matters here since the export is a default.